pkg/server/options/options.go

```diff
@@ -77,16 +80,18 @@ func NewOptions() *Options {
 		AdminAuthentication: *NewAdminAuthentication(),
 
 		Extra: ExtraOptions{
-			RootDirectory:         ".kcp",
-			ProfilerAddress:       "",
-			ShardKubeconfigFile:   "",
-			EnableSharding:        false,
-			DiscoveryPollInterval: 60 * time.Second,
+			RootDirectory:            ".kcp",
+			ProfilerAddress:          "",
+			ShardKubeconfigFile:      "",
+			EnableSharding:           false,
+			DiscoveryPollInterval:    60 * time.Second,
+			ExperimentalBindFreePort: false,
 		},
 	}
 
 	// override all the stuff
 	o.GenericControlPlane.SecureServing.ServerCert.CertDirectory = "" // will be completed to be relative to root dir
+	o.GenericControlPlane.SecureServing.Required = false
```

Comment:
does this have any implication, defaulting to false?

Reply:
no, it doesn't. It would change the flag description which we don't want. Hence, I don't set it earlier. Maybe worth a go doc.

Reply:
added comment